main.go

```diff
@@ -46,7 +49,15 @@ type Form struct {
 	Password string `json:"password"`
 }
 
-func SignUpHandler(w http.ResponseWriter, r *http.Request) {
+type DB struct {
+	Db *gorm.DB
```

Comment:
```suggestion
	DB *gorm.DB
```

https://gist.github.com/knsh14/0507b98c6b62959011ba9e4c310cd15d#initialisms